### ISSUE TITLE
Flow typing error in RegularRule

### ISSUE THREAD

**Issue by bigslycat** [cssinjs/jss#507]
After update Flow from **0.46.0** to **v0.47.0**:
```
node_modules/jss/lib/plugins/RegularRule.js.flow:40
 40:     else if (generateClassName) this.className = generateClassName(this, options.sheet)
                                                                              ^^^^^^^^^^^^^ unused function argument
   12: export type generateClassName = (rule: Rule) => string
                                       ^^^^^^^^^^^^^^^^^^^^^^ function type expects no more than 1 argument. See: node_modules/jss/lib/types.js.flow:12
```

**Comment by kof** [comment]
Thanks, looking into it.

**Comment by kof** [comment]
fixed in 7.1.3
